tap_clickup/streams.py

```diff
@@ -185,31 +211,38 @@ class FolderCustomFieldsStream(ClickUpStream):
     schema_filepath = SCHEMAS_DIR / "custom_field.json"
     records_jsonpath = "$.fields[*]"
     parent_stream_type = FolderListsStream
+    #TODO not clear why this is needed
+    partitions = None
```

Comment:
Partitions get populated from the parent stream I'm pretty sure if it's not overridden. Not certain though.